apps/full-site-editing/full-site-editing-plugin/donations/fetch-status.js

```diff
@@ -4,10 +4,14 @@
  */
 import apiFetch from '@wordpress/api-fetch';
 
-const fetchStatus = async () => {
+const fetchStatus = async ( type = null ) => {
+	let path = '/wpcom/v2/memberships/status';
+	if ( type ) {
+		path += `?type=${ type }`;
```

Comment:
Optional: just noting we can use `addQueryArgs` here:

```
import { addQueryArgs } from '@wordpress/url';

const path = addQueryArgs( '/wpcom/v2/memberships/status', { type } );
```

Reply:
`addQueryArgs` will add the param even if it's `null` though, which is an invalid param for the endpoint (resulting in an error). I could still have used it within the code block, but then it seemed silly to pull in a module just for that... so I've left it as-is.